src/kvstore/Part.cpp

```diff
@@ -441,6 +423,23 @@ bool Part::preProcessLog(LogID logId,
     return true;
 }
 
+ResultCode Part::toResultCode(raftex::AppendLogResult res) {
```

Comment:
Could make static.

Reply:
It can't be static, because we need idStr_.